core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -240,8 +240,8 @@ protected TransportTracer getTransportTracer() {
      * #listenerClosed} because there may still be messages buffered to deliver to the application.
      */
     private boolean statusReported;
-    private Metadata trailers;
-    private Status trailerStatus;
+    /** True if the status to report (set via {@link #transportReportStatus}) is OK. */
+    private boolean statusToReportIsOk;
```

Comment:
Why is this "to" report. `statusReportedIsOk`?

Reply:
Done